### DISCUSSION TITLE
Drop either `-vvv` or `--debug`

### DISCUSSION THREAD

**Discussion by ThomasLandauer** [Codeception/Codeception#6081] [Ideas]
I never understood the difference between `-v/vv/vvv` and `--debug`. At https://github.com/Codeception/Codeception/issues/4989#issuecomment-392282993 @DavertMik says `-vv` is the same as `--debug`, but I can't even see a difference between `-vv` and `-vvv`. And the output of `-v` looks the same as without any `v` to me...

Besides, having two flags for the same, leads to the unanswerable question: Which will win if you do:
```bash
vendor/bin/codecept run --debug -v
```

So I'm suggesting: **Decide for one and drop (i.e. deprecate) the other.**

Without thinking it through thoroughly, I'd say: Go for `--debug`. Why?
* Usually tests aren't that long, that there are really *three* verbosity levels required. Just show the normal output without `--debug`, and the *maximum* output with `--debug`.
* The `--debug` syntax is more compatible with related flags like `--steps` or `--silent`

**Comment by DavertMik** [comment]
`-vvv` is taken from Symfony console (it can't be removed)
`--debug` is our alias to `-vv`

Usually, you don't need `-vvv`, this is required only when the bug is in Codeception itself and you need to have to trace error that deep.
Also, this may be useful in unit & integration tests, when Codeception code gets coupled with application code, so one can affect other.

So I would say that there is purpose of them. 

But for browser or REST APi tests you probably don't need `-vvv`

**Comment by ThomasLandauer** [comment]
Well, then let's drop `--debug` and `--silent`, see https://github.com/Codeception/Codeception/pull/6082

**Reply by DavertMik** [reply]
Why?
I don't understand reasons why should we drop useful and readable alias

**Reply by ThomasLandauer** [reply]
Because having several options that all do (more or less) the same is confusing. But if you do want to keep them, I can suggest a "compromise":
* Is `--debug` really an exact alias for `-vv`? => Then I'll explain it as that.
* There is a slight difference between `--silent` and `--quiet` (one is outputting an additional line). If you remove this difference, `--silent` can be explained as alias for `--quiet`.

**Reply by DavertMik** [reply]
1. yes, please explain that `-vv` and `--debug` are the same. I like both keywords, they have some important semenatics in names
2 `--silent` and `--quiet` are different for some historical reasons, but I can't remember out what the actual difference is :cry:

**Reply by ThomasLandauer** [reply]
See https://github.com/Codeception/Codeception/pull/6147
